### ISSUE TITLE
Doesn't seem to run without git

### ISSUE THREAD

**Issue by natew** [folke/ultra-runner#87]
We're testing out ultra-runner for our docker builds, but when I run it says:

```
$ ultra -r build
❯ @dish/auth at packages/auth
❯ @dish/tscc-spec at packages/tscc-spec
❯ @dish/tsickle-loader at packages/tsickle-loader
❯ @dish/worker at packages/worker
error Not a Git repository /app/packages/auth
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Service 'jwt-server' failed to build: The command '/bin/sh -c yarn build' returned a non-zero code: 1
##[error]Process completed with exit code 1.
```

I'm guessing ultra-runner does some git check and gets upset here. Seems like that should be optional.

**Comment by folke** [comment]
Git is being used to determine changed files.

It's fairly easy to add an option to check for changed files, but rewriting the changed file detection without using Git is not that straightforward.

Would skipping file checks be enough for your use case?

**Comment by natew** [comment]
Yea, I'm not sure what file checks are doing but in this case skipping would be fine.

**Comment by folke** [comment]
The checks for file changes are used to determine if a build needs to happen, or if it can be skipped. Without these checks, builds would always behave as if being run with `--rebuild`

**Comment by folke** [comment]
Fixed in v3.4.1. No additional options are needed. A warning will be displayed when build change detection is disabled.
